src/pretix/presale/signals.py

```diff
@@ -196,6 +210,22 @@
 As with all plugin signals, the ``sender`` keyword argument will contain the event.
 """
 
+question_form_fields_overrides = EventPluginSignal(
+    providing_args=["position", "request"]
+)
+"""
+This signals allows you to override fields of the questions form that is presented during checkout
+and by default only asks for the questions configured in the backend. You are supposed to return a
+dictionary of dictionaries with globally unique keys. The value-dictionary should contain one or
+more of the following keys: ``initial``, ``disabled``. The key of the dictionary should not be the
+question's form field name (``question_n``) but rather the questions ``identifier``.
+
+The ``position`` keyword argument will contain a ``CartPosition`` object.
```

Comment:
```suggestion
The ``position`` keyword argument will contain a ``CartPosition`` or ``OrderPosition`` object.
```

It's always a cart for now, but let's document it this way, it will give us more options going forward.